internal/db/db_trace.go

```diff
@@ -11,10 +12,37 @@ import (
 type dbTracer struct{}
 
 func (dt *dbTracer) TraceQueryStart(ctx context.Context, conn *pgx.Conn, data pgx.TraceQueryStartData) context.Context {
-	logrus.WithContext(ctx).Debugf("Executing SQL with args %v: %s", data.Args, data.SQL)
+	logrus.Debug(formatSqlLog(data))
```

Comment:
don't we want to log to the context logger? :thinking:

Reply:
Good catch.